lib/manager/pipenv/extract.js

```diff
@@ -43,11 +44,31 @@ function extractFromSection(pipfile, section, registryUrls) {
     return [];
   }
   const specifierRegex = new RegExp(`^${specifierPattern}$`);
-  const deps = Object.entries(pipfile[section])
+  const oneSection = pipfile[section];
+
+  if (section === 'packages') {
```

Comment:
Why is this only applicable for 'packages' and not 'dev-packages'?

Reply:
Agreed.